tests/UrlTest.php

```diff
@@ -0,0 +1,175 @@
+<?php
+
+/*
+ * This file is part of the webmozart/path-util package.
+ *
+ * (c) Bernhard Schussek <bschussek@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Webmozart\PathUtil\Tests;
+
+use Webmozart\PathUtil\Url;
+
+/**
+ * @since  2.2
+ *
+ * @author Bernhard Schussek <bschussek@gmail.com>
+ * @author Claudio Zizza <claudio@budgegeria.de>
+ */
+class UrlTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @dataProvider provideMakeRelativeTests
+     * @covers Webmozart\PathUtil\Url
+     */
+    public function testMakeRelative($absoluteUrl, $baseUrl, $relativePath)
+    {
+        $relative = Url::makeRelative($absoluteUrl, $baseUrl);
+        $this->assertSame($relativePath, $relative);
+    }
+
+    /**
+     * @dataProvider provideMakeRelativeTests
+     * @covers Webmozart\PathUtil\Url
+     */
+    public function testMakeRelativeWithUrl($absoluteUrl, $baseUrl, $relativePath)
+    {
+        $relative = Url::makeRelative($this->createUrl($absoluteUrl), $this->createUrl($baseUrl));
```

Comment:
@webmozart So the relative paths should be removed from the data provider? That helper function had the only purpose to prepend the host only to absolute paths.

``` php
            // already relative
            array('css/style.css', '/webmozart/puli', 'css/style.css'),

            // both relative
            array('css/style.css', 'webmozart/puli', '../../css/style.css'),

            // relative to empty
            array('css/style.css', '', 'css/style.css'),
```
As you stated in one of the following comments below: "The Url class should only deal with URLs, not with paths."